test/idle-source-e2e/idle_source_test.go

```diff
@@ -13,11 +13,26 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/* Test the functionality of progressing watermark in case of idling.
+for example: publishing the data to only one replica instead of multiples.
+Once "threshold" reached to 5s(configurable) and if source is found as idle, then it will increment the watermark by
+3s(configurable) after waiting for stepInterval of 2s(configurable).
+*/
+
+//go:generate kubectl -n numaflow-system delete statefulset zookeeper kafka-broker --ignore-not-found=true
+//go:generate kubectl apply -k ../../config/apps/kafka -n numaflow-system
+// Wait for zookeeper to come up
+//go:generate sleep 60
```

Comment:
can we remove these lines?

Reply:
No, it's required to deploy kafka before running the e2e test